easypermissions/src/main/java/pub/devrel/easypermissions/AppSettingsDialogHolderActivity.java

```diff
@@ -0,0 +1,40 @@
+package pub.devrel.easypermissions;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.RestrictTo;
+import android.support.v7.app.AppCompatActivity;
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+public class AppSettingsDialogHolderActivity extends AppCompatActivity implements DialogInterface.OnClickListener {
```

Comment:
@samtstern Do you know of an other way to do this? Here's where I'm stuck: we need to be able to deliver results back to the original caller even if the activity was rotated. I believe using a holder activity is the best way to do it even considering the performance implications.